superset-frontend/src/dashboard/components/SliceHeader/index.tsx

```diff
@@ -104,9 +104,18 @@ const SliceHeader: FC<SliceHeaderProps> = ({
     [crossFilterValue],
   );
 
+  const handleClickTitle =
+    !editMode && supersetCanExplore ? onExploreChart : undefined;
+
   useEffect(() => {
     const headerElement = headerRef.current;
-    if (
+    if (handleClickTitle) {
+      setHeaderTooltip(
+        sliceName
+          ? t('Click to edit %s in a new tab', sliceName)
+          : t('Click to edit chart in a new tab'),
```

Comment:
I'm kind of surprised by `sliceName` being optional:
<img width="611" alt="image" src="https://user-images.githubusercontent.com/33317356/169573939-b6b98b97-dfb9-4f53-a726-6a82b5bb43b6.png">
Not a blocker, but would be interesting to see if it really should be optional.

Reply:
I think those checks are for the edge case where sliceName is an empty string (not sure if it's possible to achieve such state)